system-addon/content/activity-stream.html

```diff
@@ -12,6 +12,6 @@
     <script src="resource://activity-stream/vendor/react-intl.js"></script>
     <script src="resource://activity-stream/vendor/redux.js"></script>
     <script src="resource://activity-stream/vendor/react-redux.js"></script>
-    <script src="resource://activity-stream/data/content/activity-stream.bundle.js"></script>
+    <script src="chrome://activity-stream/content/activity-stream.bundle.js"></script>
```

Comment:
So now we have an odd mix of `chrome` and `resource` URIs. Couldn't this actually all be resource URIs? And only have this file (`activity-stream.html`) as content to get it loaded via `chrome:`?

This means the `css` is loaded via `resource:` and all the related/relative assets can stay where they are?